nucypher/characters/lawful.py

```diff
@@ -1209,7 +1210,7 @@ def run(self,
         # if learning:  # TODO: Include learning startup here with the rest of the services?
```

Comment:
@jMyles - May I draw your attention here :point_up_2:

Reply:
So drawn.  How can I help?

Reply:
The TODO says it all